parsl/executors/high_throughput/interchange.py

```diff
@@ -127,7 +131,7 @@ def __init__(self,
 
         self.heartbeat_thresh = heartbeat_period * 2
 
-    def get_tasks(self, count, socks):
+    def _get_tasks(self, count, socks):
```

Comment:
please pick two better names rather than using the same name twice

(this is not critical for functionality)

Reply:
Removed dead code.